js/controller.js

```diff
@@ -146,6 +151,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       $("#" + elementId + " .player_skin").css("z-index", OO.CSS.ALICE_SKIN_Z_INDEX);
       this.state.mainVideoElement = $("#" + elementId + " .video");
 
+      this.state.playerHeight = $("#" + elementId + " .innerWrapper").height();
```

Comment:
AFAIK, innerWrapper is not made of react, so it is fine.
player_skin is the one we need to be careful of not mixing.